sdk/storage/inc/blobs/internal/protocol/blob_rest_client.hpp

```diff
@@ -831,8 +830,14 @@ namespace Azure { namespace Storage { namespace Blobs {
       {
         auto request = Azure::Core::Http::Request(Azure::Core::Http::HttpMethod::Get, url);
         request.AddHeader("Content-Length", "0");
-        request.AddHeader("x-ms-version", options.Version);
-        request.AddHeader("x-ms-date", options.Date);
+        if (!options.Version.empty())
```

Comment:
Remove the version option.  The SDK is versioned to the specific version of the Service based on the swagger.

Reply:
I vaguely remember that we've talked about this version thing months back in weekly meetings. We are going to let the users change the default version if they want, but we don't guarantee everything is going to work if they do so. Now there are two methods, 1. with pipeline. 2. version field in each request options. Personally i prefer 1.